app/env.py

```diff
@@ -31,3 +31,4 @@
 
 # comma-separated list:
 ALLOWED_HOSTS = os.environ.get("ALLOWED_HOSTS", "").split(",")
+CORS_ALLOWED_ORIGINS = os.environ.get("CORS_ALLOWED_ORIGINS", "").split(",")
```

Comment:
when CORS_ALLOWED_ORIGINS isn't set it defaults to `[""]`, which [causes an error](https://github.com/CenterForOpenScience/gravyvalet/actions/runs/9598391564/job/26479890621?pr=69#step:8:23) 
```suggestion
CORS_ALLOWED_ORIGINS = tuple(filter(bool, os.environ.get("CORS_ALLOWED_ORIGINS", "").split(",")))
```
(i guess ALLOWED_HOSTS is not validated as strictly, but could do with the same fix)

Reply:
done